### DISCUSSION TITLE
HubSpot connector not updating "archived" records

### DISCUSSION THREAD

**Discussion by urkle** [airbytehq/airbyte#37347] [Connector Questions]
We setup the hubspot sync in our airbyte setup at the end of last month so we can see how well it works syncing our data.  

Today in checking the validity of our companies table I noticed that I had several records that are very out-of-date and not being updated (showing duplicates of a UNIQUE property). 

Upon investigating these dups I found that the companies with the dup property are both set as archived in HubSpot (and have the same value for that property, as HubSpot only enforces for non-archived records).   But the records are not marked as archived in our databasse and..  In one example the 2 records were archived on the 8th and 9thof April (7-8 days ago). However we have had several syncs since then in airbyte and they have not been updated to reflect that new change, they both have updated at dates before the archival.

We were hoping that airbyte would not have this issue (currently we are utilizing Segment to sync data and they have the same issue!)

**Comment by urkle** [comment]
I have reached out to HubSpot through support channels asking them about the LACK of API support to query newly archived records.

**Comment by zigamilek** [comment]
I've had this problem since forever as well!

@urkle did you find out anything?

**Reply by urkle** [reply]
@zigamilek I received largely canned response of 
```
I will make sure the product team is aware of this reason why I have proactively flagged it to them.

While we don't have an immediate solution available, please know that your feedback has been passed over.
```
Here is the "idea" post tracking this functionality. 

https://community.hubspot.com/t5/HubSpot-Ideas/Get-Archived-objects-using-search-API/idc-p/961315#M176704

[a comment by an automated account is not shown]
